Patch 1

BeauJBurroughs opened the pull request:
added index2.php and form2.css

BeauJBurroughs committed:
index.php

This is used in my environment for gate control. Pressing Open triggers the relay to pull, opening the gate and holding it open. Pressing "Open" again, releases the relay, allowing a weight switch to trigger the gate to close.  The css shows the state the relay is in by changing the color if relay is closed.  Pressing Close jogs the relay.

BeauJBurroughs committed:
Update index2.php

BeauJBurroughs committed:
form2.css

css form index2.php

BeauJBurroughs committed:
relay.py

python implementation

cryxli commented:
Hi Beau

- Please have a look at the /scripts/ directory. Wouldn't your python script fit there? (You could also add it to the existing readme :-)
- The debug echo at the top of index2.php may mess with the setCookies later on. Is it still needed?
- Does your index2.php really want to POST to index.php? (I may have confused myself by the earlier typo you've already corrected.)

Thank for considering adding to sr201!

Cheers,
cryxli

BeauJBurroughs committed:
Update index2.php

BeauJBurroughs committed:
Create relay.py

BeauJBurroughs committed:
Delete relay.py

BeauJBurroughs committed:
Update README.md

BeauJBurroughs committed:
Update README.md

BeauJBurroughs commented:
Hi. thanks for your feedback. i went ahead and made some of the changes you
suggested. i moved the python script to the scripts folder under
python-config-scripts and added a line to the readme. and fixed the typo in
the index2.php so it posts back to itself index2.php. this is just meant as
an alternative to the original index.php. as for the debug echo at the top,
i commented it out. it does not seem to affect the cookies. I also
commented out some of the other echo's in the elseif's. if you have any
more suggestions please let me know. Thank you.

Very respectfully
-Beau

On Thu, Apr 4, 2019 at 6:23 PM Urs P. Stettler <notifications@github.com>
wrote:

> Hi Beau
>
>    - Please have a look at the /scripts/ directory. Wouldn't your python
>    script fit there? (You could also add it to the existing readme :-)
>    - The debug echo at the top of index2.php may mess with the setCookies
>    later on. Is it still needed?
>    - Does your index2.php really want to POST to index.php? (I may have
>    confused myself by the earlier typo you've already corrected.)
>
> Thank for considering adding to sr201!
>
> Cheers,
> cryxli
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cryxli/sr201/pull/10#issuecomment-480085102>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AG6Xn3uo_XGxKvcgGFn001vW3NqDlGh_ks5vdntJgaJpZM4cdgD4>
> .
>
